recipes/cvxopt/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "1.1.8" %}
+
+{% set variant = "openblas" %}
+
+
+package:
+  name: cvxopt
+  version: {{ version }}
+
+source:
+  fn: cvxopt-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/c/cvxopt/cvxopt-{{ version }}.tar.gz
+  md5: d4ff0a0e51aba93f838a70302a088eea
+
+build:
+  skip: true  # [win]
+  number: 200
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.18*
+    - gsl
+    - fftw
+    - glpk
+
+  run:
+    - python
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.18*
+    - gsl
+    - fftw
+    - glpk
+
+test:
+  imports:
+    - cvxopt
+
+about:
+  home: http://cvxopt.org
+  license: GPL 3
+  summary: Convex optimization package
+
+extra:
+  recipe-maintainers:
+    - jakirkham
+    - jjhelmus
```

Comment:
If you decide you don't want to be on `cvxopt`, @jjhelmus, please let me know and I can remove you.